samples/drivers/espi/src/main.c

```diff
@@ -33,6 +33,10 @@ LOG_MODULE_DECLARE(espi, CONFIG_ESPI_LOG_LEVEL);
 
 /* 20 MHz */
 #define MIN_ESPI_FREQ         20u
+/* 25 MHz */
+#define NON_DEFAULT_FREQ      25u
+/* 66 MHz */
+#define MAX_ESPI_FREQ         66u
```

Comment:
I think changing these `*_FREQ` macros to actual frequency numbers would be better, e.g. `ESPI_FREQ_20MHZ` and `ESPI_FREQ_25MHZ`. This would make it a lot clear when reading the struct below, without the need to scroll back to the top of the file to find what `MIN_ESPI_FREQ` is.